common.blocks/menu-item/menu-item.js

```diff
@@ -31,6 +31,17 @@ provide(BEMDOM.decl(this.name, /** @lends menu-item.prototype */{
             'true' : function() {
                 this.__base.apply(this, arguments);
                 this.delMod('hovered');
+                this.domElem.attr('aria-disabled', 'true');
+            },
+            '' : function() {
+                this.__base.apply(this, arguments);
+                this.domElem.removeAttr('aria-disabled');
+            }
+        },
+        'checked' : {
```

Comment:
It's weird to write code in the different manner within one file, don't you think so?